rewards/utils/tree_utils.py

```diff
@@ -37,14 +36,14 @@ def calc_next_cycle_range(
 
     last_claim_end = tree_manager.last_publish_end_block()
     start_block = last_claim_end + 1
-
-    # Claim at last synced block
-    end_block = last_synced_block(chain)
-    if chain == Network.Arbitrum:
-        end_block = end_block - 100
-
-    # Sanity check: Ensure start block is not too far in the past
+    synced_block = last_synced_block(chain)
+    metadata = get_claimable_metadata(chain, synced_block)
+    print(metadata)
+    print(start_block)
```

Comment:
Do we need these print statements?  If so, maybe make the logging more clear as to what is being displayed